catalog/ui/catalog-ui-search/src/main/webapp/component/search-form/search-form-collection.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
We should update the filename to `search-form.collection.js`.

Reply:
done